core/src/main/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingService.scala

```diff
@@ -202,33 +203,36 @@ class SpendReportingService(userInfo: UserInfo, dataSource: SlickDataSource, big
     }
   }
 
+  case class AggregationKeyQueryField(bigQueryFieldOpt: Option[String], aliasOpt: Option[String]) {
+    def aliased(): String = (bigQueryFieldOpt, aliasOpt) match {
+      case (Some(bq), Some(key)) => s""",
```

Comment:
Why is it formatted this way?

Reply:
This was from copying it out of the original multiline query, I'll remove the extra whitespace